src/agent/onefuzz-agent/src/tasks/coverage/recorder.rs

```diff
@@ -138,18 +138,13 @@ impl CoverageRecorder {
 
         // Unless users explicitly set symbol path, don't perform external symbols
         // lookup during sancov extraction
-        if !self.config.target_env.contains_key("_NT_SYMBOL_PATH") {
-            cmd.arg("-sins");
-        }
-
         let mut target_env = self.config.target_env.clone();
-
+        let cache_path = format!("cache*{}", self.config.target_exe.parent().ok()?.display());
```

Comment:
The cache will create directories `exe` and `dll`. Those are unlikely to conflict with user provided files/directories, but it might be safer and more idiomatic to put the symbols in a `symbols` directory under the target_exe directory.